extension/src/popup/Router.tsx

```diff
@@ -244,9 +244,9 @@ export const Router = () => {
     <HashRouter>
       <RouteListener />
       <Switch>
-        <PublicKeyRoute exact path={ROUTES.account}>
+        <PrivateKeyRoute exact path={ROUTES.account}>
```

Comment:
side effect of the /account view now needing to sign txns for the user for sep10 token, so the /account page now needs the private key active

really only changes when user switches accounts. I didn't find it very annoying while testing

Reply:
Ah good catch - let's keep an eye on if this becomes a problem